foundations/installations/computer_basics.md

```diff
@@ -0,0 +1,42 @@
+### Introduction
+
+Needless to say, whilst learning to code, you'll be spending most of your time on a computer. Hence, understanding how to work with your computer is crucial. In this lesson, we'll introduce a resource which will quickly get you up to speed with some computer basics. 
+
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+* Computer software and hardware
+* How to take screenshots
+* The importance of strong passwords
+
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+Goodwill Community Foundation (GCF) resources are excellent to get you acquainted with the basics. They have accompanying videos and illustrations and are easy to read. The following readings should be sufficient, although you are welcome to explore more of GCF's resources.
+
+1. [What is a computer?](https://edu.gcfglobal.org/en/computerbasics/what-is-a-computer/1/)
+2. [What is an operating system?](https://edu.gcfglobal.org/en/computerbasics/understanding-operating-systems/1/)
+3. [What is an application?](https://edu.gcfglobal.org/en/computerbasics/understanding-applications/1/)
+4. [Open source vs closed source software](https://edu.gcfglobal.org/en/basic-computer-skills/open-source-vs-closed-source-software/1/)
+5. [Taking screenshots](https://edu.gcfglobal.org/en/techsavvy/taking-screenshots/1/)
+6. [Creating strong passwords](https://edu.gcfglobal.org/en/techsavvy/password-tips/1/)
+
+</div>
+
+
+### Knowledge Check
+
+This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.
+
+* <a class="knowledge-check-link" href="https://edu.gcfglobal.org/en/computerbasics/understanding-operating-systems/1/">Is Windows an operating system or an application?</a>
+* <a class="knowledge-check-link" href="https://edu.gcfglobal.org/en/basic-computer-skills/open-source-vs-closed-source-software/1/">What's the difference between open source and closed source software?</a>
+* <a class="knowledge-check-link" href="https://edu.gcfglobal.org/en/techsavvy/taking-screenshots/1/">What are some use cases of a screenshot?</a>
+* <a class="knowledge-check-link" href="https://edu.gcfglobal.org/en/techsavvy/password-tips/1/">Give an example of a weak and a strong password.</a>
+
+### Additional Resources
+
+This section contains helpful links to related content. It isn’t required, so consider it supplemental.
+
+* In case of disabilities and/or other restrictions, checkout [GCF's page on "What are accessibility features"](https://edu.gcfglobal.org/en/computerbasics/using-accessibility-features/1/) to learn how you can leverage these features to use your computer efficiently.
```

Comment:
```suggestion
* Checkout [GCF's page on "What are accessibility features"](https://edu.gcfglobal.org/en/computerbasics/using-accessibility-features/1/) to learn how you can leverage these features to use your computer efficiently.
```
**nitpick:** I might remove this preface since anyone might benefit from this resource.